recipes-kernel/linux/linux-xlnx_%.bbappend

```diff
@@ -17,6 +17,12 @@ SRC_URI_append_uz7ev-evcc-hdmi-v = " \
                 file://0003-drm_atomic_helper-Supress-vblank-timeout-warning-mes.patch \
 "
 
+SRC_URI_append_uz7ev-evcc-hdmi-v-n = " \
```

Comment:
as uz7ev-evcc-hdmi-v-n is an override to uz7ev-evcc-hdmi-v
Adding patch 0002 and 0003 is not needed, only nvme.cfg needs to be appended